container-host-files/etc/scf/config/role-manifest.yml

```
@@ -1968,7 +1968,7 @@ configuration:
     properties.router.balancing_algorithm: '"((ROUTER_BALANCING_ALGORITHM))"'
     properties.router.client_cert_validation: '"((ROUTER_CLIENT_CERT_VALIDATION))"'
     # We use mysql because it comes up without any other dependencies and its critical for the cluster
-    properties.router.dns_health_check_host: 'mysql-set.((KUBERNETES_NAMESPACE)).svc.((KUBERNETES_CLUSTER_DOMAIN))'
+    properties.router.dns_health_check_host: 'mysql-set((#KUBERNETES_NAMESPACE))/((KUBERNETES_NAMESPACE))'
```

```suggestion
    properties.router.dns_health_check_host: 'mysql-set((#KUBERNETES_NAMESPACE))((/KUBERNETES_NAMESPACE))'
```

Not sure what the point of an empty conditional is here though.